Bottom: adac731cdbe772d70ab85cf93d8dba176bf2978f
Top:    b1e5ff5046dcede1d63c71c8e960c2b1796dbafb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:29 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 85bffb4..37340d4 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2707,7 +2707,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
